client/my-sites/themes/auto-loading-homepage-modal.jsx

```diff
@@ -152,135 +128,66 @@ class AutoLoadingHomepageModal extends Component {
 			return null;
 		}
 
-		const {
-			name: themeName,
-			id: themeId,
-			stylesheet,
-			screenshot: themeScreenshot,
-		} = this.props.theme;
-
-		const iframeSrcKeepHomepage = `//${ this.props.siteDomain }?theme=${ encodeURIComponent(
-			stylesheet
-		) }&hide_banners=true&preview_overlay=true`;
+		const { name: themeName, id: themeId } = this.props.theme;
 
 		return (
 			<Dialog
 				className="themes__auto-loading-homepage-modal"
 				isVisible={ isVisible }
-				buttons={ [
-					{
-						action: 'keepCurrentTheme',
-						label: translate( 'Keep my current theme' ),
-						isPrimary: false,
-						onClick: this.closeModalHandler( 'keepCurrentTheme' ),
-					},
-					{
-						action: 'activeTheme',
-						label: translate( 'Activate %(themeName)s', { args: { themeName } } ),
-						isPrimary: true,
-						onClick: this.closeModalHandler( 'activeTheme' ),
-					},
-				] }
 				onClose={ this.closeModalHandler( 'dismiss' ) }
 			>
-				<Gridicon
-					icon="cross"
-					className="themes__auto-loading-homepage-modal-close-icon"
-					onClick={ this.closeModalHandler( 'dismiss' ) }
-				/>
 				<TrackComponentView
 					eventName="calypso_theme_autoloading_homepage_modal_view"
 					eventProperties={ { theme: themeId } }
 				/>
+				<Button
+					className="themes__auto-loading-homepage-modal-close-icon"
+					borderless
+					onClick={ this.closeModalHandler( 'dismiss' ) }
+				>
+					<Gridicon icon="cross" size={ 12 } />
+					<ScreenReaderText>{ translate( 'Close modal' ) }</ScreenReaderText>
+				</Button>
 				<div className="themes__theme-preview-wrapper">
-					<h1>
-						{ translate( 'How would you like to use %(themeName)s?', {
+					<h1 className="auto-loading-homepage-modal__heading">
+						{ translate( 'Activate %(themeName)s', {
 							args: { themeName },
 						} ) }
 					</h1>
-					<div className="themes__theme-preview-items">
-						<div className="themes__theme-preview-item themes__theme-preview-item-iframe-container">
-							<FormLabel>
-								<div className="themes__iframe-wrapper">
-									<Spinner />
-									{ ! isNarrow && (
-										<iframe
-											scrolling="no"
-											loading="lazy"
-											title={ translate( 'Preview of current homepage with new theme applied' ) }
-											src={ iframeSrcKeepHomepage }
+					<p className="auto-loading-homepage-modal__description">
+						{ translate(
+							'After activation, this layout will replace your existing homepage. But you can still access your old content. {{a}}Learn more{{/a}}.',
```

Comment:
[Translation has already been completed](https://github.com/Automattic/wp-calypso/pull/77475#issuecomment-1583150168) for the copy in this PR. 📝